content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -156,11 +156,15 @@ kubeadm config images pull
 You can pass `--config` to the above commands with a [kubeadm configuration file](#config-file)
 to control the `kubernetesVersion` and `imageRepository` fields.
 
-All default `k8s.gcr.io` images that kubeadm requires support multiple architectures.
+All default `registry.k8s.io` images that kubeadm requires support multiple architectures.
 
 ### Using custom images {#custom-images}
 
-By default, kubeadm pulls images from `k8s.gcr.io`. If the
+{{< note >}}
+For Kubernetes v1.22.0 through to v1.22.16, kubeadm used images from `k8s.gcr.io` by default. See [registry.k8s.io: faster, cheaper and Generally Available](/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/).
```

Comment:
D'oh. This should be a link to https://k8s.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/ and _not_ the shortened version we usually ask for.

Reply:
#38469 to resolve